app/layout.tsx

```diff
@@ -13,8 +13,8 @@ const geistMono = Geist_Mono({
 });
 
 export const metadata: Metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
+  title: "PawRide - Uber for Dogs",
+  description: "Premium transportation service for your furry friends",
```

Comment:
your description is missing some spice 🔥 add oomph like 'PawRide: Taking your furry friends on adventures they'll never fur-get!' 🐾